StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1009ClosingParenthesisMustBeSpacedCorrectly.cs

```diff
@@ -162,6 +162,18 @@ private static void HandleCloseParenToken(SyntaxTreeAnalysisContext context, Syn
                 break;
             }
 
+            foreach (var trivia in token.TrailingTrivia)
+            {
+                if ((trivia.IsKind(SyntaxKind.SingleLineCommentTrivia) || trivia.IsKind(SyntaxKind.MultiLineCommentTrivia))
```

Comment:
:question: Does this work? If so it would improve performance by not needing to call `GetLineSpan()`.

```csharp
foreach (var trivia in token.TrailingTrivia)
{
  if (trivia.IsKind(SyntaxKind.EndOfLine))
  {
    break;
  }
  else if (trivia.IsKind(SyntaxKind.SingleLineCommentTrivia)
    || trivia.IsKind(SyntaxKind.MultiLineCommentTrivia))
  {
    lastInLine = false;
    precedesStickyCharacter = false;
    break;
  }
}
```